core/src/main/java/org/hibernate/ogm/compensation/impl/InvocationCollectingGridDialect.java

```diff
@@ -97,24 +98,14 @@ public void executeBatch(OperationsQueue queue) {
 			while ( operation != null ) {
```

Comment:
With this patch, batching is not always enabled. And I'm still not convinced it's a good idea to always enable it. I'm working on Redis JSON and Ehcache to see how it goes. I'll decide what's best from there.